test/README.md

```diff
@@ -21,49 +21,47 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
 -->
 
 # How-to Run Tests
-Prerequisite:
-perl 5.10.1 or above with JSON and Text::CSV module installed; make 3.81 or above (recommend make 4.1 or above on windows); wget 1.14 or above.
+Prerequisites:
 
-Let's create the following scenario:
-For the purpose of this example, you have an OpenJ9/Java9 build in an /openjdk directory on a Linux machine and you would like to run the OpenJ9 tests against it.  Currently, the tests reside in a separate repo, so you will first clone the test repo.  Let's assume you cloned them into /mytests/test in this example, here are the commands you'd run:
-
-         cd /mytests
-         git clone git@github.ibm.com:runtimes/test
+  * perl 5.10.1 or above with JSON and Text::CSV module installed
+  * make 3.81 or above (recommend make 4.1 or above on windows)
+  * wget 1.14 or above
+  * ant 1.7.1 or above with ant-contrib.jar
 
-Complete Tasks 1 *Configure Environment* and 2 *Compile tests*, described in general terms in the *Tasks in OpenJ9* section below.  Let's assume you have ant installed on the test machine.  Here are the specific commands you'd run:
+Let's create the following scenario:
+For the purpose of this example, we assume you have an OpenJ9 SDK ready for testing. Below are the specific commands you'd run to compile and run tests. Details are explained in *Tasks in OpenJ9 Test* section below.
 
-         cd /mytests/test/TestConfig
-         export JAVA_BIN=/openjdk/bin       // this location changes depending on where the SDK you wish to test is located
-         export SPEC=linux_x86-64_cmprssptrs
-         export JCL_VERSION=latest
+         cd openj9/test/TestConfig
+         export JAVA_BIN=build/<platform>/images/jre/bin
+         export SPEC=linux_x86-64
          make -f run_configure.mk
+         make getdependency
          make test
 
 # Tasks in OpenJ9 Test
-0. Configure environment:
-  * required environment variables, dependent jars and default values
+1. Configure environment:
+  * required environment variables
+
+            JAVA_BIN=build/<platform>/images/jre/bin (this location changes depending on where the SDK you wish to test is located)
```

Comment:
Some of these sections don't read well on the web due to tabbed in text not line wrapping.  Can the lines be shorted to ensure that they are readable?  The text in section 5 & 6 have similar issues.